goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/View/ViewOffer.java

```diff
@@ -0,0 +1,26 @@
+package com.goodpeople.gooddeeds.View;
+
+import android.os.Bundle;
+
+
+import android.widget.TextView;
+
+import com.goodpeople.gooddeeds.Model.Entities.IDeed;
+import com.goodpeople.gooddeeds.R;
+
+
+public class ViewOffer extends ViewTemplate {
+    private IDeed deed;
```

Comment:
You don't need to have this here. Maybe just use it as a local variable in the onCreate method to loosen the coupling?